### ISSUE TITLE
Add ability to add file-global tags

### ISSUE THREAD

**Issue by eyeinsky** [eyeinsky/org-anki#21]
.. so one could specify a tag once and for all for every note in a file, e.g `ANKI_TAGS: russian` for an org file only for Russian vocabulary.

**Comment by gsingh93** [comment]
Just FYI, there exists a way to add an org tag to every headline: https://orgmode.org/manual/Tag-Inheritance.html

This package doesn't seem to pick it up though.

**Comment by nmlc** [comment]
Tags inheritance would be really usefull.

Currently I sync my notes like this:
```
* subject
  ** anki-entry :subject-tag:
  ** anki-entry :subject-tag:
  ** anki-entry :subject-tag:
```
If tags inheritance was supported, it would simplify things for me:
```
* subject :subject-tag:
  ** anki-entry
  ** anki-entry
  ** anki-entry
```

**Comment by eyeinsky** [comment]
- I've made tag inheritance the default and bumped major version number to 1.0.0 since this might be a breaking change to some. (Having it on by default makes sense to most people I think) 
- also added a customizable variable `org-anki-inherit-tags` to turn inheritance off for those who want it the old way
- also added a file global `ANKI_TAGS` to set tags for all entries in the file. This is used also when inheritance is turned off.

Closing this because local testing showed the feature to be working well. :)
